roles/cluster_issuer/defaults/main.yml

```diff
@@ -15,6 +15,8 @@
 cluster_issuer_name: "{{ atmosphere_ingress_cluster_issuer }}"
 cluster_issuer_type: acme
 
+cluster_issuer_acme_private_ca: false
```

Comment:
I think if we end up defaulting this to the way we discussed in #1185 then we'll need to change this.